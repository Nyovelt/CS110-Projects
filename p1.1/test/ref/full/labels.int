beq t0 t1 label2
