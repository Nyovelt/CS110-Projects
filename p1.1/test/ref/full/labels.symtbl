0	label1
0	label3
4	label2
