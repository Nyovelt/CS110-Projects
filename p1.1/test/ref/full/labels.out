.text
00628263

.symbol
0	label1
0	label3
4	label2
